src/components/ui/LogoHeader/LogoHeader.styles.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
```

Comment:
this is the figma design:
<img width="158" alt="Screenshot 2024-09-26 at 10 24 47" src="https://github.com/user-attachments/assets/72825524-c2e6-48bb-a660-34ac581e7b20">

This is your design:
<img width="113" alt="Screenshot 2024-09-26 at 10 25 13" src="https://github.com/user-attachments/assets/1a9b2a5b-b529-496f-a96f-224dcf4447ef">

Do you see the difference? The Margin between the logo and points are not the same etc. Please make it pixel perfect. But a little big bigger.